ublox-short-range/src/client.rs

```diff
@@ -55,9 +55,9 @@ pub enum DNSState {
 
 #[derive(PartialEq, Clone)]
 pub struct SecurityCredentials {
-    pub ca_cert_name: Option<heapless::String<16usize>>,
-    pub c_cert_name: Option<heapless::String<16usize>>, //TODO: Make &str with lifetime
-    pub c_key_name: Option<heapless::String<16usize>>,
+    pub ca_cert_name: Option<heapless::String<16_usize>>,
+    pub c_cert_name: Option<heapless::String<16_usize>>, //TODO: Make &str with lifetime
```

Comment:
You don't need the `_usize` on all of these.. `16` should suffice.

Reply:
I removed them.